client/data/agency-dashboard/use-fetch-dashboard-sites.ts

```diff
@@ -45,6 +53,7 @@ const useFetchDashboardSites = (
 					...( currentPage && { page: currentPage } ),
 					...agencyDashboardFilterToQueryObject( filter ),
 					...agencyDashboardSortToQueryObject( sort ),
+					per_page: per_page ?? null,
```

Comment:
Perhaps we should pass the per_page default if `null`/`undefined`?

Reply:
I had that initially, my only reservation was that if the backend default changes, we won't necessarily know to change it here. But then, this all may be somewhat more irrelevant when looking at a new API and implementation for A4A. And it's the same issue for `totalPages` in `SitesDataViews` anyway :D So I'll change it.